usmqe/web/application/implementations/web_ui.py

```diff
@@ -55,13 +57,21 @@ def go(self, _tries=3, *args, **kwargs):
         :return: view instance if class attribute ``VIEW`` is set or ``None``
             otherwise
         """
-        super(TendrlNavigateStep, self).go(_tries=_tries, *args, **kwargs)
+        try:
+            super(TendrlNavigateStep, self).go(_tries=_tries, *args, **kwargs)
+        except NavigationTriesExceeded:
+            now = datetime.strftime(datetime.now(), "%y_%m_%d_%H:%M")
+            self.view.browser.selenium.get_screenshot_as_file("screenshots/pre_nav" + now + ".png")
```

Comment:
Do we need to reraise NavigationTriesExceeded exception after taking a screenshot?